plugins/org.python.pydev.shared_core/src/org/python/pydev/shared_core/actions/LineCommentAction.java

```diff
@@ -43,11 +43,15 @@ public FastStringBuffer commentLines(String selectedText) {
         }
 
         for (String line : ret) {
+            FastStringBuffer lineBuf = new FastStringBuffer(line.length());
```

Comment:
The idea is reusing the `FastStringBuffer` as much as possible to avoid realocating memory, so, create it out of the for and then do `lineBuf.clear()` as the first thing inside the for to reuse it.